WireDomain/Sources/WireDomain/Event Decryption/ProteusMessageDecryptor.swift

```diff
@@ -95,25 +88,13 @@ struct ProteusMessageDecryptor: ProteusMessageDecryptorProtocol {
             throw ProteusMessageDecryptorError.senderFailedToEncrypt
         }
 
-        guard
-            ciphertext.count <= maxCiphertextSize,
-            let ciphertextData = Data(base64Encoded: ciphertext)
-        else {
-            throw ProteusError.decodeError
```

Comment:
This error isn't exposed anymore, and anyway it feels wrong that the client is using this error.